src/h5web/toolbar/controls/DomainSlider/DomainTooltip.module.css

```diff
@@ -12,13 +12,15 @@
 .tooltipInner {
   composes: popupInner from '../../Toolbar.module.css';
   padding: 1rem 0.375rem 1rem 0.75rem;
+  display: flex;
+  align-items: center;
```

Comment:
One more thing: I'm confused about the HTML/CSS layout. You measure `tooltipControls` and give the histogram's SVG the same size. Why do you need `align-items: center` here and why do you need an additional flex `container` inside `DataHistogram` with a `2rem` padding? I feel like this could be simplified.

Reply:
The `2rem` padding was to make space for the axes that overflow from the `svg`. But I can indeed remove the container and put a margin of `2rem` to the `svg`.

Well spotted :owl: !